hunts/models.py

```diff
@@ -133,3 +137,59 @@ def get_minutes_per_solve(self, recent=False):
 
         minutes_elapsed = (interval_end - interval_start).total_seconds() / 60
         return "{:.2f}".format(round(minutes_elapsed / solved, 2))
+
+
+class HuntSettings(models.Model):
+    hunt = models.OneToOneField(
+        Hunt,
+        on_delete=models.CASCADE,
+        related_name="settings",
+    )
+
+    #
+    # Google-specific settings
+    #
+
+    # The id of your Google Drive folder
+    # This should be part of the URL (https://drive.google.com/drive/folders/<folder_id>)
+    google_drive_folder_id = models.CharField(max_length=128, blank=True)
+
+    # The id of your Google Sheets template file
+    # This should be part of the url (https://docs.google.com/spreadsheets/d/<sheet_id>)
+    google_sheets_template_file_id = models.CharField(max_length=128, blank=True)
```

Comment:
Right now we're just using google_drive_folder_id for figuring out who can log in. New sheets are created in the same folder that the template is in. There is technically nothing enforcing that these are the same folder, so that's another future TODO I think.

Reply:
Yeah that's a good point, added a TODO.

Reply:
Oh wow, I never realized that. Totally thought `GOOGLE_DRIVE_HUNT_FOLDER_ID` was where new puzzle sheets were created

Reply:
Filed https://github.com/cardinalitypuzzles/cardboard/issues/541 for this. Can we updated the TODO to TODO(#541)?

Reply:
Thanks, done.